config/locales/de.yml

```diff
@@ -1498,9 +1503,15 @@ de:
       invitation:
         ending: Dankeschön und viel Spaß,
         greeting: Dein Pageflow-Team
-        instruction: Willkommen bei Pageflow. Bitte lege über den folgenden Link ein Kennwort fest. In Zukunft kannst Du Dich dann mit Deiner E-Mail-Adresse anmelden.
+        instruction: Willkommen bei Pageflow. Please use the following link to confirm your membership.
         salutation: Hallo %{first_name}
         subject: Ihre Pageflow Einladung
+      invitation_signin:
+        ending: Dankeschön und viel Spaß,
+        greeting: Dein Pageflow-Team
+        instruction: Willkommen bei Pageflow. Bitte lege über den folgenden Link ein Kennwort fest. In Zukunft kannst Du Dich dann mit Deiner E-Mail-Adresse anmelden.
+        salutation: Hallo %{first_name}
+        subject: Anleitung um dein Passwort zurückzusetzen
```

Comment:
These files are auto generated. Please follow the instructions in [Adding and Deleting Translation Keys](https://github.com/codevise/pageflow/wiki/Adding-and-Deleting-Translation-Keys)